diff --git a/arch/arm64/boot/dts/rockchip/px30-evb.dts b/arch/arm64/boot/dts/rockchip/px30-evb.dts
index c1bbd555f5f5..4e179a2feed5 100644
--- a/arch/arm64/boot/dts/rockchip/px30-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/px30-evb.dts
@@ -20,7 +20,7 @@ aliases {
 	};
 
 	chosen {
-		stdout-path = "serial5:115200n8";
+		stdout-path = "serial2:115200n8";
 	};
 
 	adc-keys {
@@ -614,7 +614,8 @@ &uart1 {
 	status = "okay";
 };
 
-&uart5 {
+&uart2 {
+	pinctrl-0 = <&uart2m1_xfer>;
 	status = "okay";
 };
 
